src/client/pages/Forever/useRedeemCode.ts

```diff
@@ -55,6 +56,7 @@ export const useRedeemCode = () => {
 
       switch (result.data?.redeemCodeV2.__typename) {
         case 'SuccessfulRedeemResult':
+          CampaignCode.save(code)
```

Comment:
[Question]
How the old flow works? I mean, on the new one the campaign code is get from a cookie. But, as far as I understood, in the old one the code is present int the URL until [PreOnboardingScreen](https://github.com/HedvigInsurance/web-onboarding/blob/master/src/client/pages/Forever/components/Intro/PreOnboardingScreen.tsx#L56) is rendered. But it seems we do nothing with it when redirecting PreOnboardingScreen -> Oboarding page. 🤔

Reply:
Hmm let's see if I can explain it:

1. Add the campaign code to the session
2. Show `PreOnboardingScreen`
3. Show `Onboarding` page (code already added to session)

And I've just made sure to also save it to the cookie so we can pick it up in quote cart flows when Embark loads.

Makes sense?

Reply:
Absolutely. Thank you